sdk/pr/RawResolver.go

```diff
@@ -2,9 +2,9 @@ package pr
 
 import "github.com/open-policy-agent/opa/rego"
 
-type RawResolver struct {
+type RawMapper struct {
```

Comment:
I originally thought there might be more mappers we wanted to include, but since we are just going to include the raw mapper, I would agree that we not put it into it's own package.  I've changed this.